src/snowflake/connector/constants.py

```diff
@@ -428,3 +428,9 @@ class IterUnit(Enum):
 
 
 _DOMAIN_NAME_MAP = {_DEFAULT_HOSTNAME_TLD: "GLOBAL", _CHINA_HOSTNAME_TLD: "CHINA"}
+
+_CONNECTIVITY_ERR_MSG = (
+    "Verify that the hostnames and port numbers in SYSTEM$ALLOWLIST are added to your firewall's allowed list."
+    "\nTo further troubleshoot your connection you may reference the following article: "
+    "https://community.snowflake.com/s/article/Snowflake-Client-Connectivity-Troubleshooting."
```

Comment:
Don't we want to point to the new doc? https://docs.snowflake.com/en/user-guide/client-connectivity-troubleshooting/error-messages#jdbc-errors

Reply:
thanks for the  new doc link! I'm not aware of this before